logos/dano/dano.svg

```diff
@@ -0,0 +1 @@
McfVDsHLKMw9Z9w36rd65wffSdFgMTdgo5zmSTaVxc9e2qX6vmTSTpTygmzuj1Y/pscybWPQ0A0rtYl0Lb93aNlXxy1fLXcOxS8qGACvRD21+KxFLgbmHj+pE4THQfluRm2JlF78hOvAvXV4idFEJldBvkQyTyOg1ofZ53M03QTZsOgHFZ7nWNu6QrZBdlModztgA7QVu5tjWQ2WVzmd8Fot3Rn+EcFrMnQ5dUBDEWZ/j/M0Lia7Lip1GU9f8QDAqVpnN66ZiYYiMtUyLOODNeynYNld7RwBDazwItBr7shySTmN6P3s6TgRCmcb5qYvc49wJfdM38MaimaX0OvjhuCPgGlYdhdx3YrSleERdOvxo3GLili27R+WvY3uqWvIM/Wd7Y8ToVAID+8LtzSvcjRduJKuNItG2oxX350od7gG0M1QeYFqJFJdNj+yNvLs/NmDvX7/k7KI2PLvwSbKfQk5otxv2+bdQA0Cr2Ae1aR8MU2c3wUsUSUzZqsVglPz8s7b+c0C6n5Znu6GDZqR8FEBWo6OiHHgusDxFVFIBnpGZGbPcVI0k1x9LKIjr3cz/xND3qsGtaTIZjHM/EYaL7UCxWACPxpMea87EPoH+YShrTk1uMDszlAG2PufIccDHhSiel+phL3acUfiSaqW4WVXw66WGy5X4RMdojXi99UAIxxWSMXmHd8RebWr/1iwBQtMcgsvUUc7djYuE3XE2wiBFoMYiJJQGv9oFSSDjngdQnPAHCZCCRamCq/ptsidODrEQxbm/A0RWDsJ7vg769pFRt1zam+bZw027tasIuoKng4nLQjt+Gk42txKYu730X7uFH43DS/xsDv6nP7m7KHQam6zTW5o7m8Ru3D0Wpd6sx4JzuwR9X+Z5Unm9nfBgK2ZGgBSKAYq5pYWLlPyTpTKfaQjeTm8p47PtSkz3mFg+/4665oEMuI+eDAKnt9KZCr5pB6lPp/6mC+k89acpYJdbq/818d1vZm2hRv+r3KZntwc6mbHGWOfZHhbKBl8YaQPcTFW09sDI29a/YerAptXy/J8HUJ2HPVtG7S4C+qzsebwb0Q+7+yaEzjy9/TaIrAIR3UvtSOe/mEr/pYEyE+nwZPAG2PxeUfZuOK7CIdHsYIBuHf+U0UPw+4ORnapTaCUJ91WlH/sYUS3TzizFFM2wy5wu46a58h8QOACe4+QSB2XKdVlj5NviJ6GflQr7bjS0cQhxqjjtAEZk25jo/Ilz3JvWz6hh/kFUbZ0Qb9dMEC5BtJgGLcSyw6sNImm/42dzZkAMuGgnThhHmH97Y4NxN/Gfgp4RdnWK/BTed9xdwZW5ekf3r4LUF5uIEWyPfDL1d3fTslyTH/SjOvrC2WbXrZBBlgkYT7OpcM/Hqmx9A5W/hYEU+2CI80wcTOREUud0SCdsx1sMYgdgP4Q8Z/l8lT6xq3DbYPT67TJa1WDdlmYUuEt3bdTChvFmKcTKyStz2nXUZZEctALFVa7M7EmAZk9aSkCpWY5kVBcZ7v3bEWHfr0Cu76Qn4okSXZoZ9jCpnDWVCyzQZV7v8Hm9HSLSQTLHchjejjeI1HZrESGEqpsS1SlB8DgaauM0xYQGhCWknYrDZjfE7m0KyBk0NNdawSYyku5y1qdP8BELd3tgxTahK0H0HmHXcbpdPDQxWrO+ofIRy19lgcWrsZ0ZwU/j0PqiCqUxOY9DJxTAAmP0p/NQaRJM90SEdU1eUUgc4YaRs1qqtL4xXDHk3Zq8Y70/Bb+6772xbvgzfdM35o7icqF2UB3DX5KhUx9ORq3trp5d/zZtJK5Tqq3tyl22PdVY30Uhi6oWQAMxxTXGu/C+cTuqgfTgd1CPodCbGatl31hc66oHtjRtJUhBzm1GRa40rc0c56cwVo/tgM0F/7p/pKYAFgmZfsDutNj0jDLntrGKpO00Fa5TO3bSEUN/6IKeAP1rVE2bXGE8H+q6u4CG7ukfYJroPeT+3J6Ui7T8BfIAarRcEmH4qyAZyt+gmOf6dXISyyYoPVIDRRLauGDlJNKsKst8FguxgRlmMSTGFoAiPWzuCsnhb2e6YPvUudPsri3SJxDOaZ5QkJHay9GoTnM8xCuaoa8NJSgluvPiOvsz9ZxR+1PiUTwDhRpEGhcd5HncWnRk7/9z3R/C/BifxXTwTChve59TfjTEMn74IdmTx0/ExKp/JTjVf37gNL9F3XxzE2UXJIyHVts7Xh/HjkafKF0+dXwubityfp6ZnMa3P4ur9pecFE60lC8UNssJJn8vYiP5pf2DPDPRrTDdS9S10rrvk2Y5hG6R/9pHmke9INpQPQX4V5/EfC+qIhlkJIm4vqXGSnX1eubbnJkfm/OqWwyaisQVzjJPqbK9KnCc1G4ckuxpbERm/fbsS7PHT0+DtkGWZL+2PDxFFDynaho/2ZLbHHHaOvzDZ/Zod3S3yRcJ6VfwgfFztCCftLQTdnmwVFD3xZZm8alcb/+Hd5oFAXX4Vm3ZGSKa6bOEqnA68k5sxqKN/PS6p41EqzL5lCKVuU/KxGwRNySQnaWXfQEp/UQbwM9UOFKpq0q06Gn2sUhtdLPVHOM/ZGRAGyK6Doa+mTifhDXhscmeM+VagsrsYfB9HGbRHgPvrpqTXkMdlKfiL8Q2NvrJoeGOP8ie9vmFMK6xebfkReE/Rh6CaAeWsuHOqnM3hKqjGe/KUeLFgRkmSUnYZ4oLRyOsYdCokU1xRSto3nRVYydiuj423bmNaBGy78jjOgaI/1iJjTQKFDxOtsQyec05G67EFjDwbW8WlmZn0o3/p3aJTFwoE2j9oFXxZ4hmJ/ZCEJT56DADxUeMmIzoSWftwlfMnJP0FBOg2re1hr7aTNP7ero4+mrDxdRZH7+k+N9HNu7mC1jdebpNhRTcorEuFutZ0upDpnkBOXAIPZHvixV3RPpdaue8iOl94YrOSFETQCU20biDnZqOsffIGYp3bLG3NfhmWbGabPmE7wy9ATanzEArzHvZAtodsT2
```

Comment:
Hey man, mad energy with this, but this part of the code has some png in it.
[image](https://user-images.githubusercontent.com/22182946/52441866-199b4780-2b22-11e9-96d4-69e51bb4b331.png)

Kindly take out the gradient image. I think the logo is fine if it's flat without the png
![image](https://user-images.githubusercontent.com/22182946/52441963-5404e480-2b22-11e9-894a-5db9c31dbcb0.png)